01-BootstrapperShell/BootstrapperShell/BootstrapperShell.csproj

```diff
@@ -119,6 +97,17 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Prism.Unity">
+      <Version>7.2.0.1367</Version>
+    </PackageReference>
+    <PackageReference Include="System.Threading.Tasks.Extensions">
+      <Version>4.5.3</Version>
```

Comment:
This sample doesn't work without Unity.Container being referenced directly.
It may be because Bootstrapper is deprecated and it probably still depends on Unity.Container.
It throws the following exception:
System.IO.FileLoadException: 'Could not load file or assembly 'Unity.Abstractions, Version=5.11.1.0, Culture=neutral, PublicKeyToken=489b6accfaf20ef0' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)'


I have removed System.Threading.Tasks.Extensions reference.

Reply:
Ok. I have fixed the issue now. App.Config file still had the reference to the old nuget packages.